### PR TITLE
Upgrade selenium and use geckodriver to run firefox tests.

### PR DESCRIPTION
**Pull request by feanil**
The current version of selenium, firefox and 16.04 don't play well together.

### CODE REVIEW

**Comment by feanil** [comment]
@clintonb here is just the selenium change discretely from the rest of the ubuntu 16.04 changes.

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@ language: python
 python:
     - "3.5"
 
+addons:
+  firefox: "latest"
+
+
```
Comment (clintonb) [inline .travis.yml]: Nit: remove the extra new line.

File: .travis.yml [hunk 2]
```diff
@@ -19,6 +23,9 @@ before_install:
     - "export DISPLAY=:99.0"
     - "sh -e /etc/init.d/xvfb start"
     - "export DJANGO_SETTINGS_MODULE=course_discovery.settings.test"
+    - wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux32.tar.gz
+    - tar xvzf geckodriver-v0.11.1-linux32.tar.gz
```
Comment (clintonb) [inline .travis.yml]: Nit: no need for verbosity.

File: .travis.yml [hunk 3]
```diff
@@ -19,6 +23,9 @@ before_install:
     - "export DISPLAY=:99.0"
     - "sh -e /etc/init.d/xvfb start"
     - "export DJANGO_SETTINGS_MODULE=course_discovery.settings.test"
+    - wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux32.tar.gz
```
Comment (clintonb) [inline .travis.yml]: FYI: We use npm to install this package for edx/ecommerce.

I can make this improvement later.

**Review by clintonb — APPROVED** [review]

**Review by clintonb — APPROVED** [review]
Thanks for making that change. Remember to squash your commits if necessary.

**Pull request merged**